backends/arm/test/models/test_mobilenet_v2_arm.py

```diff
@@ -22,8 +22,9 @@
 
 
 class TestMobileNetV2(unittest.TestCase):
+    """Tests MobileNetV2."""
 
-    mv2 = models.mobilenetv2.mobilenet_v2(weights=MobileNet_V2_Weights)
+    mv2 = models.mobilenetv2.mobilenet_v2(weights=MobileNet_V2_Weights.DEFAULT)
```

Comment:
Any accuracy implications or we were using these already and this is making it more explicit.